### ISSUE TITLE
As a funding report user, I can view an overview page of monthly funding reports

### ISSUE THREAD

**Issue by Derekskylight** [ctoec/data-collection#1432]
## Background 
We are finally at the point where we can go on the 'offensive' towards funding reports and are able to start upon the initial stages of its development!  Here lies the first ticket where the funding report user will see an overview page of the relevant sites and the ability to go deeper into monthly reports. 

## Acceptance Criteria
- _Reports_ is now added to the navigation bar
- Title and instructional text for directions to funding report user
- Tabulated data including _Report Month, Funding Source, Site, and Status_
- Column including the ability to view prior reports or start a new report

## Images
Link to [Figma File here](https://www.figma.com/file/bryUsb6TQnrc6Xiyj3wnEX/ECE-Reporter-%E2%80%93-WIP-Designs?node-id=809%3A8813)
![Monthly Reporting 1.PNG](https://images.zenhubusercontent.com/602d78337403e44d7db5c4f3/425abd7e-437a-4ebf-b0c6-b7596ab363f6)

**Comment by JordanGuinn** [comment]
@ehuang525 @Derekskylight I thought we wanted _all_ users with site level or org level permissions to be able to generate funding reports for their various sites?

**Comment by ehuang525** [comment]
@JordanGuinn that is correct, at least until we have more sophisticated permissions abilities. The phrasing 'funding report user' might be confusing so we can just say 'providers' instead. We should specify in the ticket that providers with site permissions will be able to submit reports for said sites. Thanks for bringing that up!

**Comment by JordanGuinn** [comment]
Sounds good!  I'll update the ticket accordingly

**Comment by JordanGuinn** [comment]
@ehuang525 @Derekskylight Few more questions:

- Can we get copy for the subheader below `Funding Reports`, instead of `Some instructional text here`?
- What should clicking the `View Report` button do, exactly?  Or is that TBD at this moment?
